unyt/tests/test_unyt_array.py

```diff
@@ -2379,3 +2379,29 @@ def test_neper_bel():
         unyt_array([0, 20], "dB") ** 2
     with pytest.raises(InvalidUnitOperation):
         np.power(unyt_array([0, 20], "dB"), -2)
+
+
+def test_delta_degC():
+    t1 = 10 * degC
+    t2 = 1 * K
+    assert t1 + t2 == 11 * degC
+    with pytest.warns(FutureWarning):
+        t2 + t1
```

Comment:
Can we test that the result we get is correct?

Reply:
Added assertion